recipes/sunode/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "sunode" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/aseyboldt/sunode/archive/v{{ version }}.tar.gz
+  sha256: 69d12dc11963db3b525325c19f143be153fc59c5f6d7fca13243941565fbf4b5
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<37]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - cffi
+    - sundials >=5.3
+  run:
+    - python
+    - numpy >=1.14
+    - cffi
+    - xarray >=0.15
+    - scipy
+    - sundials >=5.3
+    - numba >=0.49
+    - typing_extensions
+    - sympy
+
+test:
+  imports:
+    - sunode
+  requires:
+    - pytest
+    - hypothesis
+    - statsmodels
+  commands:
```

Add a check that the requirements are consistent with the `setup.py` information.
```suggestion
    - statsmodels
    - pip
  commands:
    - pip check
```

---

done